docs/guides/migrate/migration-guide.md

```diff
@@ -729,6 +729,98 @@ We deprecated the `Fiber.ID` and moved it to the `zio` package and called it the
 
 ## Runtime, Platform and Executor
 
+## Unsafe Marker
+
+To run a ZIO workflow, we usually use `ZIOAppDefault` or `ZIOApp` traits. These traits provide the `run` method which will run the workflow using their default `Runtime` system. But when we want to work with a low-level API or want to integrate with a legacy code, we need to unsafely run the workflow.
+
+In ZIO 1.x, we used the `zio.Runtime.unsafeRun` method to run a ZIO workflow:
+
+```scala
```

Comment:
Would be good to use `mdoc` on these examples where possible so we test that they compile.

Reply:
I agree, but some of them are based on ZIO 1.x, one of them is just a `diff` snippet based on ZIO 2.x, some of them are just simplified definitions of the API, and one another is based on scala 3. All of these are not type-safe. 

Fortunately, the final snippet has a type-safe mdoc modifier.